nixos/modules/services/networking/nat.nix

```diff
@@ -267,13 +267,14 @@ in
       networking.firewall = mkIf config.networking.firewall.enable {
         extraCommands = setupNat;
         extraStopCommands = flushNat;
+        package = mkDefault pkgs.iptables-nftables-compat;
```

Comment:
This should be somewhere around [here](https://github.com/NixOS/nixpkgs/blob/fec543436d4d35dacde01a9e8b69a2781841f901/nixos/modules/services/networking/nftables.nix#L105) or else it will break users configurations.

Reply:
Sorry, apparently I'm not fully awake yet... Thanks, updated :')